codec.go

```diff
@@ -9,18 +9,20 @@ import (
 // Codec is the interface for encoding/decoding session data to and from a byte
 // slice for use by the session store.
 type Codec interface {
-	Encode(deadline time.Time, values map[string]interface{}) ([]byte, error)
-	Decode([]byte) (deadline time.Time, values map[string]interface{}, err error)
+	Encode(deadline time.Time, persit bool, values map[string]interface{}) ([]byte, error)
```

Comment:
Hey @lafriks, just a heads up, there's 4 typos in this file on lines 12, 13, 18, and 25.

In each instance `persist` is misspelled as `persit`.

Reply:
Fixed